### DISCUSSION TITLE
alist webdav挂载阿里云盘，infuse添加webdav可以加载目录但是无法播放内容

### DISCUSSION THREAD

**Discussion by bacontse** [AlistGo/alist#2919] [Q&A]
路由器： rax80 固件版本[384.18]   Alist - 1.4 
症状：alist webdav挂载阿里云盘，infuse添加webdav，可以加载目录列表，但是无法播放内容，infuse播放内容都是显示加载此内容发生错误。无论是播放挂载别人分享的文件夹，还是播放自己网盘里的媒体文件，都是失败。
alist运行日志：
![image](https://user-images.githubusercontent.com/69425051/210807904-302b3f0d-4abc-4a34-9f0b-c7b12f69c2ce.png)

![78bff80f330036dd696d55ac3b6fa58](https://user-images.githubusercontent.com/69425051/210806910-562910d7-92d7-4e73-9265-025b9d52db51.jpg)

![image](https://user-images.githubusercontent.com/69425051/210806440-4702f1f4-cb17-4f78-a78c-bb30fe81c6a5.png)

希望大大能解惑！感激不尽

**Comment by omgblack7** [comment]
换个播放器，nplayer测试很多都正常

**Comment by zzmalone** [comment]
同样的问题，阿里云盘目录及内容都可以看到，就是无法播放，不管是网页或者是映射到本地。希望尽快回复并给出解决方案

**Reply by STEVENTAN100** [reply]
> 同样的问题，阿里云盘目录及内容都可以看到，就是无法播放，不管是网页或者是映射到本地。希望尽快回复并给出解决方案

我也遇到相似的问题，就是有些视频可以播放声音，但没有画面，不知道是不是ArtPlayer的问题

**Reply by shuanglu1** [reply]
你这是视频编码的问题

**Comment by SakuraYuki56** [comment]
不会是用的桌面端的token吧，得用移动端的才能播视频

**Reply by cyw026** [reply]
t移动端的怎么获取？  教程里都是在浏览器复制的

**Reply by SakuraYuki56** [reply]
官方文档里就有

**Comment by cyw026** [comment]
同样的问题，阿里云盘目录及内容都可以看到，就是无法播放，不管是网页或者是映射到本地。但是直接在Alist上面能播放

**Reply by ronnie9211** [reply]
直接用阿里云盘的驱动不行么，为什么要用webdav呢

**Comment by Luckypolo2** [comment]
webdav策略改成302了么

**Comment by zipper119** [comment]
因为移动端的token是变化的
用如下方法获得token，问题解决。
https://alist.nn.ci/zh/guide/drivers/aliyundrive.html

**Reply by lzp544765181** [reply]
就是按照这个获取的，用手机端阿里云盘扫描获取token，然后复制到alist，还是出现上面这种情况

**Comment by pototazhang** [comment]
你好，我也遇到了同样的问题，请问你的解决了吗？

**Comment by LeaiFish** [comment]
我也同样的问题 用的是ailst挂载的阿里百度夸克，所有网盘都有这个问题😭搞不懂为啥
